firmware/main.cpp

```diff
@@ -0,0 +1,140 @@
+#include "mbed.h"
+#include "CAN.h"
+#include "can-ids/CAN_IDS.h"
+
+// Heartbeat message rate in milliseconds
+#define HEARTBEAT_RATE     100ms
```

Comment:
Would probably be best to move defs to a separate header file and import it to keep things consistent and make it easier for non-coders to change constants.